src/execution/fixtures/ExecuteAfm.fixtures.ts

```diff
@@ -932,13 +932,300 @@ const twoMeasuresOneDimensionResponse: Execution.IExecutionResponses = {
+            {
+                headers: [
+                    {
+                        measureGroupHeader: {
+                            items: [
+                                {
+                                    measureHeaderItem: {
+                                        name: "Sum of dst",
+                                        format: "#,##0.00",
+                                        localIdentifier: "m_size",
+                                    },
+                                },
+                                {
+                                    measureHeaderItem: {
+                                        name: "Sum of value",
+                                        format: "#,##0.00",
+                                        localIdentifier: "m_color",
+                                    },
+                                },
+                            ],
+                        },
+                    },
+                ],
+            },
+            {
+                headers: [
+                    {
+                        attributeHeader: {
+                            name: "latlon",
+                            localIdentifier: "a_location",
+                            uri: "/gdc/md/projectId/obj/1",
+                            identifier: "label.ziplatlong.latlon",
+                            formOf: {
+                                name: "GEO pushpin",
+                                uri: "/gdc/md/projectId/obj/79",
+                                identifier: "attr.ziplatlong.latlon",
+                            },
+                        },
+                    },
+                    {
+                        attributeHeader: {
+                            name: "state",
+                            localIdentifier: "a_segmentBy",
+                            uri: "/gdc/md/projectId/obj/2",
+                            identifier: "label.ziplatlong.state",
+                            formOf: {
+                                name: "state",
+                                uri: "/gdc/md/projectId/obj/71",
+                                identifier: "attr.ziplatlong.state",
+                            },
+                        },
+                    },
+                ],
+            },
+        ],
+        links: {
+            executionResult:
+                "/gdc/app/projects/projectId/executionResults/8917638730922032128?c=5b9c03529139801e7cc5f21c27cd3218&q=eAGlksFPwjAUxv%2BVpbsubJAojMR4UCRcNFE5kR66tcOZdq%2B2r7pJ%2BN9tgUWCFwnX1%2B9973u%2Fvg0x%0AQoPBR6YEmZJlgzVKwUlCSpBONZZMVxvCa6sl6x7AqAX3snTNy1TxtPuwuq3WWQs8H46At9ffIDtg%0AV66cZHpUplC8p5OhdyuYFTMplGhw%2Bbw412Kcp2LfbG9rfuP9GKKpC4fi%2FDzj3PdbpxQznQ9ybPYa%0Alve1%2Bewp0s6%2B6brxzzskf6ud3knv7wZrAYODFJDJwGxFkxWl22Tzm7Q3t8hQnEa4iPB45O0uJDw8%0AJdxv3ce9hHi4AOzR0C1NiBL%2B%2F8r9dVX%2BrBh6wHESx9kgy46YvzgVQRVxiyTA%2FIf0k0knSJhh4CsM%0AOMyaG3Ca0O0Pg1v8Kw%3D%3D%0A&offset=0%2C0&limit=1000%2C1000&dimensions=2&totals=0%2C0",
```

Comment:
Can we reduce this long url to be same as [other results](https://github.com/gooddata/gooddata-react-components/blob/188d07ec77a2bcfa0a3f0e9f4ae6167bafd23259/src/execution/fixtures/ExecuteAfm.fixtures.ts#L35)?

Reply:
updated.